client/sections.js

```diff
@@ -42,4 +42,16 @@ sections.push( {
 	enableLoggedOut: true,
 } );
 
+if ( config.isEnabled( 'hello-world' ) ) {
+	sections.push( {
+		name: 'hello-world',
+		paths: [ '/hello-world' ],
+		module: 'hello-world',
+		enableLoggedOut: true,
+		secondary: false,
+		// group: 'sites',
+		isomorphic: true,
```

Comment:
Note that this `isomorphic` flag can cause confusion because it seems be involved in all three of: single-tree rendering, SSR, and isomorphic routing. I wonder if we can eliminate it or somehow clarify its purpose as part of this project.

Reply:
I don't think we need it for single-tree rendering, if we just use `page()` route defs, and explicitly invoke `clientRender`. (The `router` helper that we get from `isomorphic` set to true is just that, but IMHO it's easier/more advisable to stick to `page()` and do everything explicitly here.)

Reply:
+1 to clarifying things tho.

Reply:
Thanks @seear and @ockham, indeed it has been little confusing for me to separate in my mind which methods & configs play role in which feature. Although docs are pretty extensive and you guys are helping a ton to clarify different parts. <3

>+1 to clarifying things tho.

It's good to try keep this in mind going forward, as it's always easy to see something more important to work on next, rather than sit down to refactor for the sake of simplification.